GA-worksheet.md

```diff
@@ -1,69 +1,74 @@
-####Name:
+####Name: Angelica Hurtado
 ####ScriptEd
 ####Topic: HTML, CSS, and JS
 
-**Directions:** Project #2 on dash.ga.co asks you to create a blog for Jeff. Although you are not creating what someone can consider a blog, you are creating an intricate webpage using HTML, CSS and some JavaScript. Three skills that you will master in your time with ScriptEd. Log into your General Assembly account on dash.ga.co walk through the second project while answering the questions below. After completing this project you will be asked to convert this page into a four page website about your self.
+**Directions:** Project #2 on dash.ga.co asks you to create a blog for Jeff Buckley. Although you are not creating what someone can consider a blog, you are creating an intricate webpage using HTML, CSS and some JavaScript. Three skills that you will master in your time with ScriptEd. Log into your General Assembly account on dash.ga.co walk through the second project while answering the questions below. After completing this project you will be asked to convert this page into a four page website about your self.
 
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+It's basically the tag the hold everything in the header.
```

Comment:
More specifically, the header tag actually holds the introductory material for the website, such as a logo and navigational links. Check it out on Buzzfeed.com. The top of the page that lets you know that you're on Buzzfeed is the header (inspect element to clarify).